SWIG/ratehelpers.i

```diff
@@ -362,7 +362,8 @@ class OISRateHelper : public RateHelper {
             Natural lookbackDays = Null<Natural>(),
             Natural lockoutDays = 0,
             bool applyObservationShift = false,
-            const ext::shared_ptr<FloatingRateCouponPricer>& pricer = {});
+            const ext::shared_ptr<FloatingRateCouponPricer>& pricer = {},
+            DateGeneration::Rule rule = DateGeneration::Backward);
```

Comment:
You can add the same also to `forDates` below, which exports the second constructor.  In that case, you'll have to pass it to the wrapped call to `make_shared`.

Reply:
for some unclear reason this seems to break the build...

Reply:
The docker images were not yet updated with the latest changes from the library.  The next rebuild should work.